### PR TITLE
update log.go to v2, update logging statements for elastisearch

### PR DESCRIPTION
**Pull request by VivianAllen**
Update log.go requirement to v2 in go.mod and imports.
Update logging statements to format that can be parsed by
elastisearch, e.g.

log.Event(ctx, string, log.FATAL, log.Error(something1), something2)
-> log.Fatal(ctx, string, something1, something2)
log.Event(ctx, string, log.ERROR log.Error(something1), something2)
-> log.Error(ctx, string, something1, something2)
log.Event(ctx, string, log.INFO, log.Error(something))
-> log.Info(ctx, string, log.FormatErrors([]error{something}))
log.Event(ctx, string, log.WARN, log.Error(something))
-> log.Warn(ctx, string, log.FormatErrors([]error{something}))

These changes needed to allow error logs to be properly viewed in
kibana.

I think this should be a release candidate (1.5.1)

### How to review

- run make test
- eyeball changes for obvious errors

This has been tested locally via a modified `dp-areas-api` logging into the local ELK stack detailed in this PR: https://github.com/ONSdigital/dp-setup/pull/1239

### Who can review

Anyone

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,8 +4,8 @@ go 1.15
 
 require (
 	github.com/ONSdigital/dp-healthcheck v1.0.5
-	github.com/ONSdigital/dp-net v1.0.9 // indirect
 	github.com/ONSdigital/log.go v1.0.1
+	github.com/ONSdigital/log.go/v2 v2.0.5 // indirect
```
Comment (daiLlew) [inline go.mod]: Same question here - is v1 still needed? If so it be good to know why its still required.

**Review by daiLlew — COMMENTED** [review]
Change looks fine but I just had 1 question about depending on v1 still.

File: go.mod (hunk 1 above)
Comment (VivianAllen) [reply go.mod]: as in https://github.com/ONSdigital/dp-authorisation/pull/13 - rookie mistake I think. I'll remove the references to v1 here.

Comment (VivianAllen) [reply go.mod]: fixed

**Review by daiLlew — APPROVED** [review]

**Pull request merged**